sora-android-sdk/src/main/kotlin/jp/shiguredo/sora/sdk/channel/rtc/RTCComponentFactory.kt

```diff
@@ -166,13 +167,31 @@ class RTCComponentFactory(
         return constraints
     }
 
-    fun createVideoManager(): RTCLocalVideoManager? =
-        mediaOption.videoCapturer?.let {
-            val videoManager = RTCLocalVideoManager(it)
+    fun createVideoManager(context: Context): RTCLocalVideoManager? {
+        // ユーザー設定の VideoCapturer がある場合はそれを使う
+        mediaOption.userSettingVideoCapturer()?.let {
+            val videoManager = RTCLocalVideoManager(it, mediaOption.soraCameraConfig)
             SoraLogger.d(TAG, "videoManager created: $videoManager")
-            videoManager
+            return videoManager
         }
 
+        // VideoCapturer が未設定かつ、soraCameraConfig が設定されている場合は
+        // SDK 内部で VideoCapturer を生成する
+        mediaOption.soraCameraConfig?.let { config ->
+            val capturer = CameraCapturerFactory.create(context, config.frontFacingFirst)
+            if (capturer == null) {
+                SoraLogger.e(TAG, "cannot create camera capturer")
```

Comment:
CameraCapturerFactory#create() 側で出しているメッセージと同じなので、同じ内容が並んでしまうのが気になる
```suggestion
                SoraLogger.e(TAG, "create RTCLocalVideoManager failed")
```
とか？

Reply:
提案ありがとうございます、これで修正します 🙏🏼